Dockerfile

```diff
@@ -1,12 +1,17 @@
 # Build stage uses a full golang image to build a statically linked binary
 FROM golang:1.12 AS builder
 WORKDIR /usr/src
+
+# Download and cache dependencies
+COPY go.mod go.sum ./
+RUN go mod download
+
 COPY . .
 
-RUN CGO_ENABLED=0 go build -a -installsuffix cgo -o bin/atlas-service-broker
+RUN ./scripts/build-production-binary.sh bin/atlas-service-broker
 
 # Run stage uses a much smaller base image to run the prebuilt binary
-FROM alpine:latest
```

Comment:
Pinning the version helps us to have more reproducible builds.

Reply:
This is good actually. Using latest might not always be the best idea, instead a specific version that we know is suitable.